docs/README-windows.md

```diff
@@ -0,0 +1,125 @@
+# Building & Installing from source in Windows platform
+```
+last updated: 2020-10-10
+tested at: 2020-10-09
+```
+
+## Prerequisite
+
+The steps are tested under this envionment. Please prepare these installed.
+
+```
+Windows 10 (64 bit)
+Visual Studio Community 2017
+Cmake 3.18.3
+OpenCV 3.4.11
+libwebp 1.1.0 windows-x64
+```
+
+- Download Visual Studio Community 2017, please visit [here](https://visualstudio.microsoft.com/vs/older-downloads/).
+- Download Cmake 3.18.3, please visit [here](https://cmake.org/download/).
+- Download OpenCV 3.4.11, please visit [here](https://opencv.org/releases/).
+- Download libwebp 1.1.0, please visit [here](https://storage.googleapis.com/downloads.webmproject.org/releases/webp/index.html).
+
+
+## Notes on install Cmake
+
+In order to run Cmake in `CMD`, please make sure `PATH envionment variable` contains the folder of Cmake. eg: `C:\Program Files\CMake\bin`
+
+One may check this by by running `echo %PATH%` in CMD.
+
+
+## Notes on install OpenCV
+
+The folder location of OpenCV is important. This program depends on this library.
+The downloaded OpenCV file is usually an archive (self-extracting archive). Please extract to a certain folder. eg: `D:\opencv` (i.e. the `opencv extrated folder`)
+
+Copy files listed below from folder `D:\opencv\build\x64\vc15\bin` to `System32 folder`. One may need Administrator right of the Windows system to do so.
+```
+opencv_ffmpeg3411_64.dll
+opencv_world3411.dll
+opencv_world3411.pdb
+opencv_world3411d.dll
+opencv_world3411d.pdb
+```
+
+**Alternatively**, one may add the path of OpenCV binary (eg: `D:\opencv\build\x64\vc15\bin`) to `PATH envionment variable`. You may refer to this [web page](https://www.deciphertechnic.com/install-opencv-with-visual-studio/).
+
+
+
+## Notes on install libwebp
+
+It locates within `Repo folder`, eg: `D:\splitwebp\3rdparty\libwebp\lib\windows-x64`
```

Comment:
Instead of calling it `Repo folder`, wouldn't it be specific call it something like "`splitwebp` root directory"